### PR TITLE
Render/vote

### PR DESCRIPTION
**Pull request by noahnovickf**

### CODE REVIEW

File: src/containers/post/post.component.jsx [hunk 1]
```diff
@@ -1,7 +1,14 @@
-import React from "react";
+import React, { useState } from "react";
 import { voteForOption } from "database/votePost";
 
 const Post = (props) => {
+  const [voteView, setVoteView] = useState(false);
+  //console.log(props);
+
+  const handleVote = (obj) => {
```
Comment (lolcode8) [inline src/containers/post/post.component.jsx:8]: Can have a better variable name than `obj`. Always best to have meaningful variables names for code readability

File: src/containers/post/post.component.jsx [hunk 2]
```diff
@@ -1,7 +1,14 @@
-import React from "react";
+import React, { useState } from "react";
 import { voteForOption } from "database/votePost";
 
 const Post = (props) => {
+  const [voteView, setVoteView] = useState(false);
+  //console.log(props);
```
Comment (lolcode8) [inline src/containers/post/post.component.jsx:6]: Cleanup

File: src/containers/post/post.component.jsx [hunk 3]
```diff
@@ -1,7 +1,14 @@
-import React from "react";
+import React, { useState } from "react";
 import { voteForOption } from "database/votePost";
 
 const Post = (props) => {
+  const [voteView, setVoteView] = useState(false);
```
Comment (lolcode8) [inline src/containers/post/post.component.jsx:5]: Maybe better to name this `canUserViewVote`?

File: src/containers/post/post.component.jsx [hunk 4]
```diff
@@ -10,20 +17,36 @@ const Post = (props) => {
...
+          {props.optionAName}: {props.optionA + 1}
+        </div>
+        <div
+          className={`${
+            voteView ? "show" : "hidden"
+          } bg-gray-300 hover:bg-gray-400 w-20 h-10 rounded-r-lg`}
+        >
+          {props.optionBName}: {props.optionB + 1}
+        </div>
+      </div>
```
Comment (lolcode8) [inline src/containers/post/post.component.jsx:49]: Not going to block this for now, but this is an example of repeated code that can turned into a function such as `renderVoteOption` and takes in a couple of arguments and decides whether to render optionA info or optionB info

Comment (noahnovickf) [reply src/containers/post/post.component.jsx:49]: We want both of them rendering though

Comment (lolcode8) [reply src/containers/post/post.component.jsx:49]: Yeah can render both by calling the function twice with different parameters

**Comment by lolcode8** [comment]
Can we close this PR now?

**Pull request merged**